src/main/java/seedu/bankwithus/BankWithUs.java

```diff
@@ -83,4 +84,14 @@ public static void main(String[] args) {
             return;
         }
     }
+
+    public  void createAccount() {
+        System.out.println("Whats your name?");
+        ui.createScanner();
+        String userName = ui.getNextLine();
+        System.out.println("How much would you like to add as Balance?");
+        float balance = Float.parseFloat(ui.getNextLine());
```

Comment:
Make sure to handle the NumberFormatException

Reply:
Thanks for the heads up, Noted.